backend/api/tests/contract-tests/tests/testClientRegistryContract.spec.ts

```diff
@@ -73,61 +23,121 @@ describe("ClientRegistryProvider", () => {
     });
 
     describe("When a POST request is made to create a client", () => {
-        test("it should return a message", async () => {
+        const postEvent: RegisterClientPayload = {
+            contactEmail: "pacttest.account@digital.cabinet-office.gov.uk",
+            service: {
+                serviceName: "My test service",
+                id: "service#testRandomId"
+            }
+        };
+
+        const client_config = {
+            client_name: postEvent.service.serviceName,
+            public_key: public_key,
+            redirect_uris: ["http://localhost/"],
+            contacts: [postEvent.contactEmail],
+            scopes: ["openid", "email", "phone"],
+            subject_type: "pairwise",
+            service_type: "MANDATORY",
+            sector_identifier_uri: "http://gov.uk",
+            client_locs: ["P2"]
```

Comment:
client locs are not returned in the response body - is this okay ?

Reply:
Ah yeah sort of remember finding this -  https://govukverify.atlassian.net/browse/ATO-1040. We can fix this as part of the above ticket

Reply:
Yep remember coming across this a while ago: https://govukverify.atlassian.net/browse/ATO-1040 - we can fix this as part of this ticket